{"name": "test_login_error[item2]", "status": "passed", "parameters": [{"name": "item", "value": "{'username': '888888', 'pwd': '5466544', 'code': '123456', 'excepted': '抱歉，账号或密码错误！'}"}], "start": 1560320454525, "stop": 1560320456862, "uuid": "052ba76c-0b94-4260-9a97-5aee493f9c89", "historyId": "4d61c4440e2fe708de54238acd3e621d", "fullName": "TestCase.test_0_login.TestLogin#test_login_error[item2]", "labels": [{"name": "tag", "value": "login"}, {"name": "parentSuite", "value": "TestCase"}, {"name": "suite", "value": "test_0_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "bj_jd-PC"}, {"name": "thread", "value": "10352-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCase.test_0_login"}]}